starlette/requests.py

```diff
@@ -249,7 +248,8 @@ async def form(self) -> FormData:
                 parse_options_header is not None
             ), "The `python-multipart` library must be installed to use form parsing."
             content_type_header = self.headers.get("Content-Type")
-            content_type, options = parse_options_header(content_type_header)
+            content_type: bytes
+            content_type, _ = parse_options_header(content_type_header)
```

Comment:
What's the problem with `parse_options_header`?

Reply:
It's untyped. It also returns a tuple and the second variable was an unused variable.